src/globus_cli/commands/streams/__init__.py

```diff
@@ -0,0 +1,43 @@
+import typing as t
+
+from globus_cli.parsing import group
+
+
+def streams_flatten_response(in_doc: dict[str, t.Any]) -> dict[str, list[t.Any]]:
+    output_dict: dict[str, list[t.Any]] = {}
+    attributes_doc = in_doc.pop("attributes")
+    for i in attributes_doc:
+        output_dict[f"attributes_{i}"] = attributes_doc.get(i)
+    for i in in_doc:
+        output_dict[i] = in_doc.get(i)
+    relationships = in_doc.get("relationships")
+    for o in relationships:
+        inner = relationships.get(o).get("data")
+        for i in inner:
+            output_dict[f"relationships_{o}_{i}"] = relationships.get(i)
```

Comment:
I'm really struggling to understand this one.

From your access pattern, my best guess is that `relationships` is a dictionary in the form:
```
{
   "key1": {"data": ["key2", "key3"]},
   "key2": {"data": ["key3"]},
   "key3": {"data": [???]
}  
```

And the function here would be building the key/value pairs
```
{
  "relationship_key1_key2": {"data": ["key3"]},
  "relationship_key1_key3": {"data": [???]},
  "relationship_key2_key3": {"data": [???]}
}
```

Am I holding that right?